kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue

```diff
@@ -14,13 +14,13 @@
       {{ $tr('welcomeModalPermissionsDescription') }}
     </p>
 
-    <span class="welcome-modal-dismiss-button-wrapper">
+    <div class="buttons welcome-modal-dismiss-button-wrapper">
```

Comment:
can we remove the `welcome-modal-dismiss-button-wrapper` class?

Reply:
I think so. I didn't want to go through the onboarding to test it. But changing it from span to div should get it to inherit that remaining `width: 100%` rule